src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-setup.mdx

```diff
@@ -46,23 +48,32 @@ Go to the repository for your language and follow the instructions to instrument
+The export relies on the [OpenTelemetry Protocol](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/otlp.md), or OTLP for short, which is a general purpose telemetry data delivery protocol designed for the OpenTelemetry project. This protocol describes how to encode and transmit telemetry data, which makes it a natural choice for data transport. Each language SDK provides an OTLP exporter you can configure to export data over OTLP.
 
-![Diagram showing OpenTelemetry using New Relic's OTLP endpoint.](./images/native_otlp.svg "Diagram of OTLP endpoint")
-<figcaption>Here's an example of sending data from your service directly to New Relic.</figcaption>
+You have two choices for exporting data to New Relic via OTLP:
 
-To complete this third step, first familiarize yourself with some required New Relic settings, and then complete the steps in the OTLP exporter documentation for your language.
+  * Directly from your app:
 
-### Review New Relic settings for exports [#review-settings]
+  ![Diagram showing a direct export to New Relic to your app](./images/native-otlp-no-collector.png "Diagram showing a direct export to New Relic to your app")
+  <figcaption>
+  The OTLP exporter in your service can export directly to the New Relic OTLP receiver.
+  </figcaption>
+  * Export from an OpenTelemetry collector:
 
-Before you go to the external OTLP exporter documentation, consult the table below so you're ready to do the following:
+  ![Diagram showing the export to New Relic from a collector](./images/native_otlp_with_collector.png "Diagram showing the export to New Relic from a collector")
+  <figcaption>
+  You can set up the collector as a gateway that receives, processes, and exports to the New Relic OTLP receiver. Alternatively, you can set up the collector as an agent that runs with your application or runs on the same host.
+</figcaption>
 
-* Configure the OTLP exporter to add a header (```api-key```) whose value is the license key for the New Relic account you want to send data to.
-* Based on your integration, configure the [endpoint](#note-endpoints) where the exporter sends data to New Relic. The endpoints are region-specific, so use the one according to where your account is based. For example, use the EU endpoint if your account is based in Europe. 
+
+Whether you export directly from your app or from a collector, you'll need to:
+
+1. Be ready to configure the OTLP exporter to add a header (```api-key```). The value is the license key for the New Relic account you want to send data to.
+2. Based on your integration, be ready configure the [endpoint](#note-endpoints) where the exporter sends data to New Relic. The endpoints are region-specific, so use the one according to where your account is based. For example, use the EU endpoint if your account is based in Europe. Use the table below as a guide and see the additional considerations about ports and endpoints:
```

Comment:
when I read this, I was wondering where the 'additional considerations' were, so maybe could link to them from there.

Reply:
@zuluecho9, I switched it up to this: "Use the table below as a guide, and see the additional considerations about [ports](#note-ports) and [endpoints](#note-endpoints) that follow:"